code/script.js

```diff
@@ -0,0 +1,170 @@
+            heroImg.style.backgroundPosition = "center"
+            heroImg.style.backgroundRepeat = "no-repeat"
+            heroImg.style.backgroundPosition = "relative"
+            heroImg.style.backgroundSize = "cover"
+            if ((i + 1) === heroImgArray.length) {
+                setTimeout(function () { heroImgSequence() }, (seconds * 1000))
+            } else { i++; }
+        }, (seconds * 1000) * i)
+    }
+}
+heroImgSequence()
+
+
+//Function to fetch and display profile information.
+const userProfile = () => {
+    fetch(USER_URL)
+        .then(res => res.json())
+        .then(data => {
+            console.log(data)
+            userContainer.innerHTML = `
+        <img src="${data.avatar_url}" class="avatar"/>
+        <p class="full-name">${data.name}</p>
+        <p class="username">Username: ${data.login}</p>
+        <p class="location">${data.location}</p>
+        <a href="${data.html_url}" target="blank"><img src="${githubIcon.src}" alt="${githubIcon.alt}"  class="github-icon"/></a>
+        <a href="${data.blog}" target="blank"><img src="${linkedinIcon.src}" alt="${linkedinIcon.alt}" class="linkedin-icon"/></a>
+        <a href="https://www.facebook.com/anna.dimitrakopoulos" target="blank"><img src="${facebookIcon.src}" alt="${facebookIcon.alt}" class="facebook-icon"/></a>
+        `
+        })
+}
+
+//Function to fetch and display my repos.
+const fetchRepos = () => {
+    fetch(MY_REPOS)
+        .then((res) => res.json())
+        .then((data) => {
+            //Clear innerHTML to prevent duplicated repos.
+            projectsContainer.innerHTML = ``
+            const forkedRepos =
+                data.filter(repo => {
+                    return repo.fork && repo.name.startsWith("project-") && selectedLanguages.includes(repo.language)
+                })
+            //Function to sort repos by latest date.
+            forkedRepos.sort(function (a, b) {
+                return new Date(b.pushed_at) - new Date(a.pushed_at)
+            })
+            forkedRepos.forEach(repo => {
+                projectsContainer.innerHTML += `
+                      <div class="repo" id=${repo.name}>
+                        <p class="forked-repos"> <img src="${repoIcon.src}" alt="${repoIcon.alt}" class="repo-icon"/> ${repo.name} </p> 
+                        <a href="${repo.html_url}" target="blank">Go to repo!</a> 
+                        <p class="forked-repos"> <img src="${branchIcon.src}" alt="${branchIcon.alt}" class="branch-icon"/> ${repo.default_branch}</p>
+                        <p class="forked-repos"> <img src="${pushIcon.src}" alt="${pushIcon.alt}" class="push-icon"/> ${new Date(repo.pushed_at).toDateString()}</p>
+                        <p id="commit-${repo.name}">Amount of commits: </p> 
+                      </div>
+                        `
+            })
+
+            fetchPullRequests(forkedRepos)
+
+            //Only draw the chart once.
```

Comment:
To prevent multiple charts to be drawn when clicking checkboxes? or to make it appear if it does not? Anyway nice that you added that to prevent multiple ones